DEPENDENCIES.rst

```diff
@@ -114,27 +114,35 @@ and (optionally):
 #. libblas-dev
 #. liblapack-dev
 #. libgoogle-perftools-dev
-#. python3
-#. python3-dev
-#. python3-tables
-#. python3-pandas
-#. python3-numpy
-#. python3-nose
-#. python3-jinja2
-#. cython3
+#. python-dev
+#. python-tables
+#. python-pandas
+#. python-numpy
+#. python-nose
+#. python-jinja2
+#. cython
```

Comment:
I am -1 on this change. This will not work with Python 3. If anything, all of the above packages are valid and should be listed.  If you want something that is guaranteed to work all of them should be installed. 

This is in addition to the fact that we should really be migrating away from Python 2 as support will end in a couple of years. It is fine to support Python 2 for the time being but I don't think we should recommend it.